scripts/india/geo/districts_test.py

```diff
@@ -58,3 +58,43 @@ def test_get_district_name_to_lgd_code_mapping(self):
         district_lgd_code = mapper.get_district_name_to_lgd_code_mapping(
             "telangana", "warangal")
         self.assertEqual(district_lgd_code, "522")
+
+        district_lgd_code = mapper.get_district_name_to_lgd_code_mapping(
+            "andhra pradesh", "nandyal")
+        self.assertEqual(district_lgd_code, "755")
+
+        district_lgd_code = mapper.get_district_name_to_lgd_code_mapping(
+            "andhra pradesh", "ntr")
+        self.assertEqual(district_lgd_code, "749")
+
+        district_lgd_code = mapper.get_district_name_to_lgd_code_mapping(
+            "Meghalaya", "eastern west khasi hills")
```

Comment:
nit: "meghalaya" with lowercase 'm'

Reply:
Its purposely done in a testcase to see if it matches regardless of Case.